luigi/scheduler.py

```diff
@@ -224,6 +224,9 @@ def prune(self, config):
         if self.last_active + config.worker_disconnect_delay < time.time():
             return True
 
+    def assistant(self):
```

Comment:
This could be a property. `worker.assistant` instead of `worker.assistant()` looks better to me :)